serialization/src/main/scala/sbt/serialization/SerializedValue.scala

```diff
@@ -20,72 +23,59 @@ sealed trait SerializedValue {
 object SerializedValue {
   def apply[V](value: V)(implicit pickler: SPickler[V]): SerializedValue =
     JsonValue[V](value)(pickler)
+
+  /** Reconstitutes a SerialziedValue from a json string. */
+  def fromJsonString(value: String): SerializedValue =
+    JsonValue.fromJsonString(value)
+  /** Reconstitutes a SerialziedValue from a json AST. */
+  def fromJsonAST(value: JValue): SerializedValue =
```

Comment:
perhaps not.  Not sure it matters.  JValue is meant to be a minimal/stable jar....